cfme/configure/about.py

```diff
@@ -3,20 +3,44 @@
 from utils import version
 
 product_assistance = Region(
-    locators={
-        'quick_start_guide': "//a[normalize-space(.)='Quick Start Guide']",
-        'insight_guide': "//a[normalize-space(.)='Insight Guide']",
-        'control_guide': "//a[normalize-space(.)='Control Guide']",
-        'lifecycle_and_automation_guide':
-        "//a[normalize-space(.)='Lifecycle and Automation Guide']",
-        'integrate_guide': {
-            version.LOWEST: "//a[normalize-space(.)='Integrate Guide']",
-            '5.3': "//a[normalize-space(.)='Integration Services Guide']",
-            '5.4': "//a[normalize-space(.)='REST API Guide']"
+    locators=version.pick({
```

Comment:
We can't version.pick here, since test runners that don't have a `base_url` (like those using sprout) will explode with an SSH failure. I recently had to [tweak Region.title](https://github.com/RedHatQE/cfme_tests/commit/2eca241253161c39541ac5502e5a5fb189a7c411) to support versioned dicts, and this PR might need similar treatment.